src/test/java/org/shikshalokam/backend/scp/TestScpPermissionCRUDOperations.java

```diff
@@ -175,33 +176,65 @@ private Response createPermission(String prefix, String module, String requestTy
         return response;
     }
 
-    @Test(description = "Verifies the getPermissions API for a valid user.")
+    @Test(priority = 7 , description = "Verifies the getPermissions API for a valid user.")
     public void testGetPermissionsWithValidPayload() {
         logger.info("Started calling the GetPermissions API with valid token.");
 
         Response response = getPermissions(true);
         response.prettyPrint();
 
         // Validate response code
-        Assert.assertEquals(response.getStatusCode(), 200, "Failed to fetch permissions.");
+        assertEquals(response.getStatusCode(), 200, "Failed to fetch permissions.");
         logger.info("Permissions retrieved successfully.");
     }
 
-    @Test(description = "Verifies the getPermissions API with an invalid token.")
+    @Test(priority = 8 , description = "Verifies the getPermissions API with an invalid token.")
     public void testGetPermissionsApiWithInvalidToken() {
         logger.info("Started calling the GetPermissions API with an invalid token.");
 
         Response response = getPermissions(false);
 
         // Validate response code for invalid token
-        Assert.assertEquals(response.getStatusCode(), 401, "Expected 401 Unauthorized error.");
+        assertEquals(response.getStatusCode(), 401, "Expected 401 Unauthorized error.");
         logger.info("Received expected error for invalid token.");
     }
+    @Test(priority = 9 , description = "Verifies the functionality of deleting user's permission by ID.")
+    public void testDeletePermissionById() {
```

Comment:
can you run this test method alone ?
this method should depend on  testCreatePermissionWithValidPayload()
Check once .